Gemfile

```diff
@@ -124,7 +124,8 @@ platforms :ruby, :mswin, :mswin64, :mingw, :x64_mingw do
   gem "sqlite3", "~> 1.3.6"
 
   group :db do
-    gem "pg", ">= 0.18.0"
+    # gem "pg", ">= 0.18.0"
+    gem "pg", ">= 1.1.0.pre", github: "larskanis/ruby-pg", branch: "lazytuple"
```

Comment:
Not sure if you're using something specific from the branch, but it looks like there's a general 1.1.3 version of `pg` available? https://rubygems.org/gems/pg/versions/1.1.3